app/src/ui/dialog/header.tsx

```diff
@@ -47,7 +47,7 @@ interface IDialogHeaderProps {
  * might be necessary to use this component directly.
  */
 export class DialogHeader extends React.Component<IDialogHeaderProps, {}> {
-  private onCloseButtonClick = (e: React.MouseEvent<HTMLAnchorElement>) => {
+  private onCloseButtonClick = () => {
```

Comment:
I _think_ we should prevent the default action of this event here now that we're turning into a button. It's _probably_ okay but I'm a bit wary given that the Dialog component is just a big `<Form />` with attached handlers for submission and resets. With the anchor tag we were safe but I'd be much more comfortable if we called preventDefault here.

Reply:
Added and tested. Makes sense.. I put a comment about it being as a preventative measure for this so that if we find it hurts something in the future, we know this was the reason.